docker/postgis_init.sh

```diff
@@ -6,10 +6,12 @@ psql -v ON_ERROR_STOP=1 --username="$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-E
     CREATE DATABASE simplified_registry_dev;
     CREATE USER simplified WITH PASSWORD 'simplified';
     GRANT ALL PRIVILEGES ON DATABASE simplified_registry_dev TO simplified;
+    ALTER DATABASE simplified_registry_dev OWNER TO simplified;
 
     CREATE DATABASE simplified_registry_test;
     CREATE USER simplified_test WITH PASSWORD 'simplified_test';
     GRANT ALL PRIVILEGES ON DATABASE simplified_registry_test TO simplified_test;
+    ALTER DATABASE simplified_registry_test OWNER TO simplified_test;
```

Comment:
These changes are due to the PG16 upgrade that we did, and could move out to a separate PR if desired.

Similar to changes for PG16 needed here: https://github.com/ThePalaceProject/hosting-playbook/pull/720